### PR TITLE
HTMLFrameMojit enhancements

### PR DESCRIPTION
**Pull request by caridy**
First commit to clean up the tests, moving it into the proper folder, adjusting the "mojito test" to execute those tests as part of the "fw" category. This is in preparation for other commits to enhance/cleanup the current HTMLFrameMojit.

Note: I will probably include the quick fix for issue #230 as well as part of this branch.

### CODE REVIEW

File: source/lib/tests/autoload/mojits/HTMLFrameMojit/tests/controller.server-tests.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
```
Comment (drewfish) [inline source/lib/tests/autoload/mojits/HTMLFrameMojit/tests/controller.server-tests.js]: To match how other tests are organized withing source/lib/tests. the controller test should go in here instead:  `source/lib/tests/autoload/mojits/HTMLFrameMojit/controller.server-tests.js`.

**Comment by caridy** [comment]
@drewfish let me change that right now.

**Comment by caridy** [comment]
@isao is this still needed after all the changes in the tests?

**Comment by isao** [comment]
ahh this never landed? bummer. @add0n is refactoring HTMLFrameMojit, maybe he can benefit from or apply this. Otherwise I can try to clean this up if you're too busy.

**Comment by add0n** [comment]
Isao - Go ahead and clean this up, if you would please. The refactoring of HTMLFrameMojit is leading into a larger discussion / refactoring of how we can enhance Mojito's client capabilities (to encourage folks to push more logic client side for better performance and user experience).

**Comment by caridy** [comment]
Closing this in favor of PR #926

**Pull request closed without merging**